flowetl/flowetl/flowetl/operators/staging/mount_event_operator_factory.py

```diff
@@ -0,0 +1,43 @@
+from airflow.operators.postgres_operator import PostgresOperator
+from flowetl.operators.staging.event_columns import event_column_mappings
+
+
+def create_mount_event_operator(*, event_type: str):
+    """
+    A class factory function that produces an Airflow operator for mounting a foreign table containing data of
+    event_type.
+    Parameters
+    ----------
+    event_type:str
```

Comment:
Should be spaces around the colon, and for a limited valid set use `{'call', 'sms'..}` instead of `str`